packages/discord.js/src/structures/VoiceChannel.js

```diff
@@ -35,6 +35,21 @@ class VoiceChannel extends BaseGuildVoiceChannel {
       permissions.has(PermissionFlagsBits.Speak, false)
     );
   }
+
+  /**
+   * Send a soundboard sound to a voice channel the user is connected to.
+   * @param {SoundboardSound} sound the sound to send
+   * @returns {void}
+   * @fires Client#voiceChannelEffectSend
```

Comment:
This is misleading though. The `@fires` tag indicates that the event might be emitted inside this method. When in reality the event will emit later, when the gateway event reaches our websocket, completely unrelated to this REST call.